db_script.py

```diff
@@ -58,49 +101,66 @@ def get_data(dbs, replicas_port=None, user_db_port=None, user=None, password=Non
-                SQL_Query = pd.read_sql_query("SELECT page_id,page_title FROM page \
-                    WHERE page_content_model='Scribunto' AND page_namespace=828", conn)
-                df_page = pd.DataFrame(SQL_Query, columns=['page_id', 'page_title']).applymap(encode_if_necessary)
+                cur.execute("USE " + db + "_p")
+                SQL_Query = pd.read_sql_query(
+                    "SELECT page_id, page_is_redirect, page_is_new FROM page "
+                    "WHERE page_content_model='Scribunto' AND page_namespace=828",
+                    conn,
+                )
+                df_page = pd.DataFrame(SQL_Query).applymap(encode_if_necessary)
 
                 # Saving to db
                 save_to_db(df_page, db, user_db_port, user, password)
-                print('Finished loading scripts from ', db)
+                # print("Finished loading scripts from ", db)
             conn.close()
         except Exception as err:
-            print('Error loading pages from db: ', db, '\nError:', err)
+            print("Error loading pages from db:", db, "\nError:", err)
 
     print("Done loading from databases.")
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description="Updates Lua scripts in database in Toolforge, fetching info from database replicas. "
-                    "To use from local PC, use flag --local and all the additional flags needed for "
-                    "establishing connection through ssh tunneling."
-                    "More help available at "
-                    "https://wikitech.wikimedia.org/wiki/Help:Toolforge/Database#SSH_tunneling_for_local_testing_which_makes_use_of_Wiki_Replica_databases"
+        "To use from local PC, provide all the additional flags needed for "
+        "establishing connection through ssh tunneling."
+        "More help available at "
+        "https://wikitech.wikimedia.org/wiki/Help:Toolforge/Database#SSH_tunneling_for_local_testing_which_makes_use_of_Wiki_Replica_databases"
+    )
+    local_data = parser.add_argument_group(
+        title="Info for connecting to Toolforge from local pc"
+    )
+    local_data.add_argument(
+        "--replicas-port",
+        "-r",
+        type=int,
+        default=None,
+        help="Port for connecting to meta table through ssh tunneling, if used.",
+    )
+    local_data.add_argument(
+        "--user-db-port",
+        "-udb",
+        type=int,
+        default=None,
+        help="Port for connecting to tables, created by user in Toolforge, "
+        "through ssh tunneling, if used.",
+    )
+    local_data.add_argument(
+        "--user", "-u", type=str, default=None, help="Toolforge username of the tool."
+    )
+    local_data.add_argument(
+        "--password",
+        "-p",
+        type=str,
+        default=None,
+        help="Toolforge password of the tool.",
     )
-    parser.add_argument("--local", "-l", action="store_true",
```

Comment:
Notice I haved removed the usage of `local` and added `default=None` to reduce a few lines of code when calling functions.